editors/liveview/assets/css/app.css

```
@@ -3,3 +3,551 @@
+   `<textarea class="bt-editor-ta">`. The two layers MUST share identical font,
+   font-size, line-height, padding and tab-size so the highlighted glyphs sit
+   exactly under the typed (transparent) glyphs — keep these rules in sync.
+   The `.tok-*` colours below resolve the `--t-*` variables themed per surface.
+   ========================================================================== */
+.bt-editor-wrap {
+  position: relative; flex: 1; min-height: 0; overflow: hidden; background: var(--code-bg);
+}
+.bt-editor-pre, .bt-editor-ta {
+  margin: 0; padding: 12px 14px; font-family: var(--code-font, ui-monospace, monospace);
+  font-size: 12.5px; line-height: 1.6; tab-size: 2; -moz-tab-size: 2;
+  white-space: pre; word-wrap: normal; border: 0;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)